src/app/donation-start/donation-start-form/donation-start-form.component.ts

```diff
@@ -1010,13 +998,30 @@ export class DonationStartFormComponent implements AfterContentChecked, AfterCon
       return;
     }
 
+    if (!this.stripeElements) {
+      console.error('Stripe Elements not ready');
+      return;
+    }
+
     // Card element is mounted the same way regardless of donation info. See
     // this.createDonationAndMaybePerson().subscribe(...) for Payment Request Button mount, which needs donation info
     // first and so happens in `preparePaymentRequestButton()`.
-    this.card = this.stripeService.getCard();
-    if (this.cardInfo && this.card) { // Ensure #cardInfo not hidden by PRB success.
-      this.card.mount(this.cardInfo.nativeElement);
-      this.card.on('change', this.cardHandler);
+    this.stripePaymentElement = this.stripeElements.create(
```

Comment:
Should we set a `paymentMethodOrder` so that e.g. wallet placement is predictable over time? I would suggest putting wallets first, then `card`.

Reply:
For now, we'll probably add a `wallets` key to turn them off (keeping PRBs as the way we support wallets), as the smallest next step.

Reply:
Wallets are now turned off.